subsys/net/lib/nrf_cloud/src/nrf_cloud_codec.c

```diff
@@ -92,6 +92,7 @@ static const char *const sensor_type_str[] = {
 #define JSON_KEY_MSGTYPE	"messageType"
 #define JSON_KEY_APPID		"appId"
 #define JSON_KEY_DATA		"data"
+#define JSON_KEY_TOPIC_BATCH	"/batch"
```

Comment:
apparently the correct topic suffix is `/bulk`

Reply:
Yeah, I completely missed that. :cry: I'll update the PR.